plugins/make_deb

```diff
@@ -239,6 +239,7 @@ EOF
         fi
         cp -R ${GIT_ROOT}/python .
         cp -R ${GIT_ROOT}/VERSION.${plugin_type}.${plugin_name} ./python/fledge/plugins/${plugin_type}/${plugin_name}
+        echo ${pkg_name} > ./python/fledge/plugins/${plugin_type}/${plugin_name}/.Package
```

Comment:
Why use this extra variable from the Package file. The script already generates the package name itself, so better to use the same generated name and avoid the duplication and potential for errors.

Reply:
There is no extra variable is being used here. See the logic here 

If plugin_package_name variable exists in Package file then used this else
https://github.com/fledge-iot/fledge-pkg/blob/8457dd4f9d1e0f2e455c7bb77a83e0981da9f206/plugins/make_deb#L123

So no impact at all.